Tests/ApodiniTests/JobsTests.swift

```diff
@@ -0,0 +1,62 @@
+// swiftlint:disable force_unwrapping force_cast
+//
+//  File.swift
+//  
+//
+//  Created by Alexander Collins on 29.12.20.
+//
+
+@testable import Apodini
+import XCTest
+
+final class JobsTests: ApodiniTests {
+    struct FailingJob: Job {
+        @_Request var request: Request
+        
+        func run() { }
+    }
+    
+    class TestJob: Job {
+        var num = 0
+        
+        func run() {
+            print("\(num)")
+        }
+    }
+    
+    struct KeyStore: ApodiniKeys {
+        var failingJob: FailingJob
+        var testJob: TestJob
+        var testJob2: TestJob
+    }
+    
+    func testFailingJobs() throws {
+        XCTAssertThrowsError(try Scheduler.shared.dequeue(\KeyStore.failingJob))
+        XCTAssertThrowsError(try Scheduler.shared.enqueue(FailingJob(), with: "* * * * *", \KeyStore.failingJob, on: app.eventLoopGroup.next()))
+        XCTAssertThrowsError(try Scheduler.shared.enqueue(TestJob(), with: "* * * *", \KeyStore.testJob, on: app.eventLoopGroup.next()))
+        XCTAssertThrowsError(try Scheduler.shared.enqueue(TestJob(), with: "A B C D E", runs: 5, \KeyStore.testJob, on: app.eventLoopGroup.next()))
+    }
+    
+    func testEnvironmentInjection() throws {
+        let scheduler = EnvironmentValues.shared.scheduler
+        let job = TestJob()
+        try scheduler.enqueue(job, with: "*/10 * * * *", \KeyStore.testJob, on: app.eventLoopGroup.next())
+        let environmentJob = EnvironmentValues.shared.values[ObjectIdentifier(\KeyStore.testJob)] as! TestJob
```

Comment:
```suggestion
        let environmentJob = try XCTUnwrap(
            EnvironmentValues.shared.values[ObjectIdentifier(\KeyStore.testJob)] as? TestJob
        )
```

Yesterday I learned: `XCTUnwrap`. Added benefit: Tests fail gracefully instead of crashing 😄 And you can remove `swiftlint:disable ...` at the top.

Reply:
This is really useful, thank you :D